### PR TITLE
Check if user needs to add token to PLEX

### PR DESCRIPTION
**Pull request by fernandog**
@Thraxis can you test this please?

https://github.com/SiCKRAGETV/sickrage-issues/issues/1756

### CODE REVIEW

File: sickbeard/notifiers/plex.py [hunk 1]
```diff
@@ -227,6 +227,12 @@ def update_library(self, ep_obj=None, host=None, username=None, password=None, p
                     logger.log(u'PLEX: Error while trying to contact Plex Media Server: ' + ex(e), logger.WARNING)
                     hosts_failed.append(cur_host)
                     continue
+                except Exception as e:
+                    if 'invalid token' in str(e):
+                        logger.log(u'PLEX: Please set TOKEN in Plex settings: ', logger.ERROR)
+                    else
```
Comment (miigotu) [inline sickbeard/notifiers/plex.py]: else:

**Comment by fernandog** [comment]
@miigotu done. squashed

**Comment by Thraxis** [comment]
@fernandog I'll try to get this tested tomorrow.  been a bit slammed RL

**Comment by miigotu** [comment]
@fernandog shouldnt this be `except ParseError as e:` ? We don't want to be catching all exceptions I think.

**Comment by fernandog** [comment]
@Thraxis working?

**Comment by fernandog** [comment]
@miigotu if we don't catch all, probably user will receive a traceback

**Pull request merged**